### ISSUE TITLE
[Feature] Support Chrome not only Chromium

### ISSUE THREAD

**Issue by universeroc** [microsoft/playwright#2367]
I'm using playwright for autotesting WebRTC and Chromium is not support H.264 default, that's the very key point and also other difference for example mp4 playing default not support in my mind.

**Comment by arjun27** [comment]
Hi @universeroc, you can download the latest Chrome Canary (from https://www.google.com/chrome/canary/) and launch it using `playwright.chromium.launch({ executablePath: PATH_TO_CANARY })`.

Let me know if this works for you.

**Comment by universeroc** [comment]
> Hi @universeroc, you can download the latest Chrome Canary (from https://www.google.com/chrome/canary/) and launch it using `playwright.chromium.launch({ executablePath: PATH_TO_CANARY })`.
> 
> Let me know if this works for you.

Yes, it works as your way and I did it by just copy all files of Chrome installed on my machine and replace the chrome-win's files so everything works, but it's a lot inconvenient. It's not **auto** enough as what playwright aims to achieve, that's the reason I opened this issue.

Also the firefox's version I checked is nightly build, maybe the lastest stable channel is better in my opinion, as almost web app should be tested through on latest stable browsers not or not only the lastest dev version.

I'm new to playwright maybe when I'm used to use playwright I'll contribute some PR for this issue

Finally, thanks for all your work, playwright is really really great, simple, powerful :)
